README.md

```diff
@@ -69,14 +72,14 @@ git clone https://github.com/louislam/uptime-kuma.git
 cd uptime-kuma
 npm run setup
 
-# Option 1. Try it
+# Option 1. Run with node.
 node server/server.js
 
-# (Recommended) Option 2. Run in the background using PM2
+# (Recommended) Option 2. Run uptime kuma in background using PM2
 # Install PM2 if you don't have it:
 npm install pm2 -g && pm2 install pm2-logrotate
 
-# Start Server
+# After installing uptime kuma (if running with PM2):
```

Comment:
```suggestion
# After installing uptime-kuma (if running with PM2):
```

Reply:
The steps are already talking about PM2, adding `(if running with PM2)` is strange in my opinion. I prefer original one.

Reply:
Actually, It is kinda confusing for me if it is the original.